API/middleware.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
```

Comment:
I don't understand how this middleware works. It looks like it'll always return the `index.html` unless the requested url contains '/api/`. How does it hand off to json-server to server static files?

Now that I think about it, given what you said (and what I read) about the way the "routes" feature works, I'm surprised we even need this middleware.